stestr/config_file.py

```diff
@@ -62,8 +62,10 @@ def get_run_command(self, options, test_ids=None, regexes=None):
             top_dir = options.top_dir
         elif self.parser.has_option('DEFAULT', 'top_dir'):
             top_dir = self.parser.get('DEFAULT', 'top_dir')
-        command = "${PYTHON:-python} -m subunit.run discover -t" \
-                  " %s %s $LISTOPT $IDOPTION" % (top_dir, test_path)
+
+        python = 'python' if sys.platform == 'win32' else '${PYTHON:-python}'
```

Comment:
Are you sure? I just looked at: https://docs.python.org/2/using/windows.html#excursus-setting-environment-variables and it seems to use it. I'm booting up a windows vm right now so I can test things, because I've got a couple other questions about the dbm stuff that I want to test